src/src/WebServer/HardwarePage.h

```diff
@@ -3,6 +3,13 @@
 
 #include "../WebServer/common.h"
 
+enum class SPI_Options_e {
```

Comment:
This enum should not be defined in the webserver code.
Bettr to place it in `src/DataTypes`, including the `toString` function.

Reply:
Yes, will move it there